__tests__/lib/createGHA.test.ts

```diff
@@ -86,7 +84,7 @@ describe('#createGHA', () => {
         expect(yamlOutput).toBeValidSchema(ghaWorkflowSchema);
         expect(yamlOutput).toMatchSnapshot();
         expect(fs.writeFileSync).toHaveBeenCalledWith(getGHAFileName(fileName), expect.any(String));
-        expect(console.info).toHaveBeenCalledTimes(1);
+        expect(consoleInfoSpy).toHaveBeenCalledTimes(1);
```

Comment:
Moved this over to the spy because it was already available and used in other tests.